.jenkins

```diff
@@ -92,13 +111,19 @@ pipeline {
                             args '-v /tmp/ccache.kokkos:/tmp/ccache'
                         }
                     }
-                    environment {
-                        OPTIONS = '--with-options=compiler_warnings --arch=Volta70 --with-cuda --with-cuda-options=enable_lambda --debug'
-                    }
                     steps {
-                        sh 'ln -s /usr/bin/ccache /usr/local/bin/g++'
                         sh 'ccache --zero-stats'
-                        sh 'rm -rf build && mkdir -p build && cd build && ../generate_makefile.bash $OPTIONS && make test -j8'
+                        sh '''rm -rf build && mkdir -p build && cd build && \
+                              cmake \
+                                -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
Does this set KOKKOS_ENABLE_DEBUG and KOKKOS_ENABLE_DEBUG_BOUNDS_CHECK etc?

Reply:
The previous --debug option does that inside the Makefile based system.

Reply:
It does not.  Changing.

Reply:
Looks good

Reply:
Shouldn't `CMAKE_BUILD_TYPE=Debug` imply `KOKKOS_ENABLE_DEBUG` and `KOKKOS_ENABLE_DEBUG_BOUNDS_CHECK`?